src/internal/operations/backup.go

```diff
@@ -937,24 +937,46 @@ func (op *BackupOperation) createBackupModels(
 
 	// Add tags to mark this backup as either assist or merge. This is used to:
 	// 1. Filter assist backups by tag during base selection process
-	// 2. Differentiate assist backups from merge backups
```

Comment:
Could you please fix the comments on line 938 & 939 above? It's out of date now.

Reply:
thanks for catching this